From d5c81cb4125d2ffbfb957ec7cd1543b33d81ee75 Mon Sep 17 00:00:00 2001
From: Philippe Gerum <rpm@xenomai.org>
Date: Sat, 12 Sep 2020 18:54:37 +0200
Subject: [PATCH 0646/1013] evl/poll: add (back) wrapper files to define
 oob_poll_wait

This reverts commit 7c81fc4feda7f114ae6721a56860ebe136a715f8.

Reinstate the dovetail/poll.h wrapper files we had to drop from the
mere Dovetail code.

Signed-off-by: Philippe Gerum <rpm@xenomai.org>
---
 arch/arm/include/dovetail/poll.h   | 7 +++++++
 arch/arm64/include/dovetail/poll.h | 7 +++++++
 arch/x86/include/dovetail/poll.h   | 7 +++++++
 3 files changed, 21 insertions(+)
 create mode 100644 arch/arm/include/dovetail/poll.h
 create mode 100644 arch/arm64/include/dovetail/poll.h
 create mode 100644 arch/x86/include/dovetail/poll.h

diff --git a/arch/arm/include/dovetail/poll.h b/arch/arm/include/dovetail/poll.h
new file mode 100644
index 000000000000..76e51be38a40
--- /dev/null
+++ b/arch/arm/include/dovetail/poll.h
@@ -0,0 +1,7 @@
+/* SPDX-License-Identifier: GPL-2.0 */
+#ifndef _EVL_DOVETAIL_POLL_H
+#define _EVL_DOVETAIL_POLL_H
+
+#include <asm-generic/evl/poll.h>
+
+#endif /* !_EVL_DOVETAIL_POLL_H */
diff --git a/arch/arm64/include/dovetail/poll.h b/arch/arm64/include/dovetail/poll.h
new file mode 100644
index 000000000000..76e51be38a40
--- /dev/null
+++ b/arch/arm64/include/dovetail/poll.h
@@ -0,0 +1,7 @@
+/* SPDX-License-Identifier: GPL-2.0 */
+#ifndef _EVL_DOVETAIL_POLL_H
+#define _EVL_DOVETAIL_POLL_H
+
+#include <asm-generic/evl/poll.h>
+
+#endif /* !_EVL_DOVETAIL_POLL_H */
diff --git a/arch/x86/include/dovetail/poll.h b/arch/x86/include/dovetail/poll.h
new file mode 100644
index 000000000000..76e51be38a40
--- /dev/null
+++ b/arch/x86/include/dovetail/poll.h
@@ -0,0 +1,7 @@
+/* SPDX-License-Identifier: GPL-2.0 */
+#ifndef _EVL_DOVETAIL_POLL_H
+#define _EVL_DOVETAIL_POLL_H
+
+#include <asm-generic/evl/poll.h>
+
+#endif /* !_EVL_DOVETAIL_POLL_H */
-- 
2.30.2

